### PR TITLE
SPSH-1183

### PR DESCRIPTION
**Pull request by YoussefBouch**
# Description

Added a new variable "pageTotal" that includes the number of orgas returned after the search criteria has been applied.

## Approval for review
- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

### CODE REVIEW

File: src/modules/organisation/persistence/organisation.repository.ts [hunk 1]
```diff
@@ -298,10 +298,10 @@ export class OrganisationRepository {
         personPermissions: PersonPermissions,
         systemrechte: RollenSystemRecht[],
         searchOptions: OrganisationSeachOptions,
-    ): Promise<Counted<Organisation<true>>> {
```
Comment (marode-cap) [inline src/modules/organisation/persistence/organisation.repository.ts:301]: Please either add a named type or at least use named tuples, as a reader I have no idea what the two numbers are supposed to represent

File: src/modules/organisation/persistence/organisation.repository.integration-spec.ts
Comment (marode-cap) [inline src/modules/organisation/persistence/organisation.repository.integration-spec.ts:1]: You've changed the repo to return more data, so there should be at least one test to actually verify that data

**Review by marode-cap — CHANGES_REQUESTED** [review]

File: src/modules/organisation/persistence/organisation.repository.integration-spec.ts
Comment (YoussefBouch) [reply src/modules/organisation/persistence/organisation.repository.integration-spec.ts:1]: Done

Comment (YoussefBouch) [reply src/modules/organisation/persistence/organisation.repository.integration-spec.ts:1]: done

File: src/modules/organisation/persistence/organisation.repository.ts (hunk 1 above)
Comment (YoussefBouch) [reply src/modules/organisation/persistence/organisation.repository.ts:301]: Using named tuple now.

**Review by marode-cap — APPROVED** [review]

**Pull request merged**